tensorflow/compiler/tf2tensorrt/utils/trt_shape_optimization_profiles.h

```diff
@@ -102,8 +109,26 @@ struct OptimizationProfileConfig {
     }
     return true;
   }
+
+ private:
+  // In dynamic shape mode the optimization profile dims are only allowed to
+  // differ from the network input dims where the network input dims have -1
+  // values. We ensure that prof_dim matches the network input (mask).
+  void applyMask(nvinfer1::Dims* prof_dim, const nvinfer1::Dims& mask) {
```

Comment:
How about renaming these:
    applyMask => accumulateInputDimensions
    prof_dim => prof_dims
    mask => input_dims

Reply:
ApplyMask does not accumulate, just enforces the compatibility condition expected by TRT, so I renamed to enforceCompatibility. Rest is applied.